src/System.Management.Automation/engine/Modules/GetModuleCommand.cs

```diff
@@ -573,9 +573,11 @@ private void GetLoadedModules(string[] names, IDictionary<string, ModuleSpecific
             IDictionary<string, ModuleSpecification> moduleSpecTable,
             PSModuleInfo module)
         {
+            const WildcardOptions options = WildcardOptions.IgnoreCase | WildcardOptions.CultureInvariant;
             foreach (ModuleSpecification moduleSpec in moduleSpecTable.Values)
             {
-                if (moduleSpec.Name == module.Name || moduleSpec.Name == module.Path || module.Path.Contains(moduleSpec.Name))
+                WildcardPattern namePattern = WildcardPattern.Get(moduleSpec.Name, options);
+                if (namePattern.IsMatch(module.Name) || moduleSpec.Name == module.Path || module.Path.Contains(moduleSpec.Name))
```

Comment:
Is path case sensitivity a problem here?

Reply:
It could be, but it is not a regression